### DISCUSSION TITLE
[Item stats] Include moonlight moth mix boost

### DISCUSSION THREAD

**Discussion by Pirocchio** [runelite/runelite#18085] [Ideas]
Include the boost from the moonlight moth mix, a flat +22 prayer points regardless of level, like the prayer potion one.
(Also, I didn't check, but maybe sunlight moth mixes are also missing??)

![image](https://github.com/user-attachments/assets/6caf49e5-2fbb-4bba-8977-b85fe7745906)
![image](https://github.com/user-attachments/assets/ce7500de-5494-42ea-a4dc-1004e8792e56)

**Comment by YvesW** [comment]
https://github.com/runelite/runelite/pull/17607
